htdocs/adherents/class/adherent.class.php

```diff
@@ -314,6 +314,11 @@ class Adherent extends CommonObject
 	 */
 	public $invoice;
 
+	/**
+	 * @var string 	Import ID
+	 */
+	public $import_key;
```

Comment:
Declare property with 
* @var ?string Import ID
to have it null possible.

But best is to remove declaration as it is already declared into CommonObject.class.php

Reply:
Good thinking, I removed it!